src/test/java/com/epam/edumanagementsystem/admin/rest/repository/SuperAdminRepositoryTest.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.edumanagementsystem.admin.rest.repository;
+
+import com.epam.edumanagementsystem.admin.model.entity.SuperAdmin;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@TestPropertySource(properties = {"compile('org.flywaydb:flyway-core')"})
+class SuperAdminRepositoryTest {
+
+    @Autowired
+    private SuperAdminRepository superAdminRepository;
+
```

Comment:
Here also you can just create one object outside of the test methods and use it.

Reply:
Thank you for the review!